style.css

```diff
@@ -0,0 +1,167 @@
+body {
+  background: #e5e5e5;
+  margin: 0;
+  padding: 0;
+  color: #484848;
+  font-size: 14px;
+  font-family: Circular, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.container {
+  position: relative;
+  width: 375px;
+  height: 674px;
+  top: 0;
+  background: #fff;
+  margin-left: auto;
+  margin-right: auto;
+  border-bottom-left-radius: 50px;
+}
+
+header {
```

Comment:
As mentioned in the [requirements for this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox%20in%20all%202%20sections.), please make sure to use CSS Flexbox to align your containers and the elements within them :frog: :+1: